doc/prelimsLinux.rst

```diff
@@ -0,0 +1,222 @@
+Preliminary Steps for Linux Distributions
+=========================================
+
+1) Install Python 3.6+
+----------------------
+Check your current python version(s) by entering the following commands into the command line: 
+
+.. code-block:: shell
+
+   python --version
+   python3 --version
+   python3.6 --version
+
+If Python 3.6+ is not present, follow these instructions depending on your operating system:
+
+Running a **Linux** distribution, Python must be compiled. Download and compile the most recent Python release with the following commands (find the link to the correct .tar file `here <https://www.python.org/downloads/>`__):
+
+.. note::
+
+   If CMake is not already on your machine see step 3a to install it.
+
+.. code-block:: shell
+   
+   wget https://www.python.org/ftp/python/3.6.1/Python-3.6.1.tar.xz
+   sudo tar xzf Python-3.6.1.tar.xz
+   cd Python-3.6.1
+   ./configure       # sudo might be required
+   make altinstall   # sudo might be required
+
+Some **newer Linux** distributions may be able to install Python 3.6+ using apt-get.
+
+.. code-block:: shell
+
+   apt-get install python3.6   # sudo might be required
+
+.. note::
+
+   It can be helpful to use an alias to aid in differentiation between Python releases. paste the following lines into your ~/.bashrc or ~/.bash_aliases file (be sure to enter `source ~/.bashrc` or restart your shell):
```

Comment:
I think this might be a bad idea since if you make these aliases, then it becomes hard to ever use the system Python (2.7). Anyway, if you use the virtual environment, it will do this for you (in a more temporary way).